Ruff check failed

```
ruff 0.5.0
warning: The `ignore-init-module-imports` option is deprecated and will be removed in a future release. Ruff's handling of imports in `__init__.py` files has been improved (in preview) and unused imports will always be flagged.
tests/test_pad_collation.py:92:12: E721 Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks
   |
90 |     @parameterized.expand(TESTS)
91 |     def test_pad_collation(self, t_type, collate_method, transform):
92 |         if t_type == dict:
   |            ^^^^^^^^^^^^^^ E721
93 |             dataset = CacheDataset(self.dict_data, transform, progress=False)
94 |         else:
   |

tests/test_pad_collation.py:107:16: E721 Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks
    |
105 |         # check collation in forward direction
106 |         for data in loader:
107 |             if t_type == dict:
    |                ^^^^^^^^^^^^^^ E721
108 |                 shapes = []
109 |                 decollated_data = decollate_batch(data)
    |

tests/test_pad_collation.py:116:12: E721 Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks
    |
114 |                 self.assertTrue(len(set(shapes)) > 1)  # inverted shapes must be different because of random xforms
115 | 
116 |         if t_type == dict:
    |            ^^^^^^^^^^^^^^ E721
117 |             batch_inverse = BatchInverseTransform(dataset.transform, loader)
118 |             for data in loader:
    |

Found 3 errors.
Check failed!
```

--- a/docs/source/transforms.rst
+++ b/docs/source/transforms.rst
@@ -667,6 +667,12 @@ Post-processing
     :members:
     :special-members: __call__
 
+`Invert`
+"""""""""
+.. autoclass:: Invert
+    :members:
+    :special-members: __call__
+
 Regularization
 ^^^^^^^^^^^^^^
 

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -38,7 +38,6 @@ exclude = "monai/bundle/__main__.py"
 
 [tool.ruff]
 line-length = 133
-lint.ignore-init-module-imports = true
 lint.ignore = ["F401", "E741"]
 
 [tool.pytype]

--- a/tests/test_pad_collation.py
+++ b/tests/test_pad_collation.py
@@ -89,7 +89,7 @@ class TestPadCollation(unittest.TestCase):
 
     @parameterized.expand(TESTS)
     def test_pad_collation(self, t_type, collate_method, transform):
-        if isinstance(t_type, dict):
+        if t_type is dict:
             dataset = CacheDataset(self.dict_data, transform, progress=False)
         else:
             dataset = _Dataset(self.list_data, self.list_labels, transform)
@@ -104,7 +104,7 @@ class TestPadCollation(unittest.TestCase):
         loader = DataLoader(dataset, batch_size=10, collate_fn=collate_method)
         # check collation in forward direction
         for data in loader:
-            if isinstance(t_type, dict):
+            if t_type is dict:
                 shapes = []
                 decollated_data = decollate_batch(data)
                 for d in decollated_data:
@@ -113,7 +113,7 @@ class TestPadCollation(unittest.TestCase):
                     self.assertTrue(len(output["image"].applied_operations), len(dataset.transform.transforms))
                 self.assertTrue(len(set(shapes)) > 1)  # inverted shapes must be different because of random xforms
 
-        if isinstance(t_type, dict):
+        if t_type is dict:
             batch_inverse = BatchInverseTransform(dataset.transform, loader)
             for data in loader:
                 output = batch_inverse(data)

--- a/tests/test_pad_collation.py
+++ b/tests/test_pad_collation.py
@@ -89,7 +89,7 @@ class TestPadCollation(unittest.TestCase):
 
     @parameterized.expand(TESTS)
     def test_pad_collation(self, t_type, collate_method, transform):
-        if t_type == dict:
+        if isinstance(t_type, dict):
             dataset = CacheDataset(self.dict_data, transform, progress=False)
         else:
             dataset = _Dataset(self.list_data, self.list_labels, transform)
@@ -104,7 +104,7 @@ class TestPadCollation(unittest.TestCase):
         loader = DataLoader(dataset, batch_size=10, collate_fn=collate_method)
         # check collation in forward direction
         for data in loader:
-            if t_type == dict:
+            if isinstance(t_type, dict):
                 shapes = []
                 decollated_data = decollate_batch(data)
                 for d in decollated_data:
@@ -113,7 +113,7 @@ class TestPadCollation(unittest.TestCase):
                     self.assertTrue(len(output["image"].applied_operations), len(dataset.transform.transforms))
                 self.assertTrue(len(set(shapes)) > 1)  # inverted shapes must be different because of random xforms
 
-        if t_type == dict:
+        if isinstance(t_type, dict):
             batch_inverse = BatchInverseTransform(dataset.transform, loader)
             for data in loader:
                 output = batch_inverse(data)

